static/css/style.scss

```diff
@@ -491,10 +470,12 @@ $primary-color: #fff;
 	max-width: 65px;
 	max-height: 27px;
 	opacity: 0.3;
+	transition: opacity 1s;
```

Comment:
Please, consider not so long timing `transition: opacity 0.2s;`

Reply:
change time of delay duration.